substrate-node/pallets/pallet-tfgrid/src/lib.rs

```diff
@@ -1081,36 +1081,37 @@ pub mod pallet {
             node_id: u32,
             node_certification: NodeCertification,
         ) -> DispatchResultWithPostInfo {
-            let account_id = ensure_signed(origin)?;
-
-            if let Some(certifiers) = AllowedNodeCertifiers::<T>::get() {
-                ensure!(
-                    certifiers.contains(&account_id),
-                    Error::<T>::NotAllowedToCertifyNode
-                );
+            // Only council/root or allowed certifiers can modify node certification
+            if !T::RestrictedOrigin::ensure_origin(origin.clone()).is_ok() {
+                let account_id = ensure_signed(origin)?;
+                if !AllowedNodeCertifiers::<T>::get()
+                    .unwrap_or(vec![])
```

Comment:
Maybe you can add a default value to AllowedNodeCertifiers so that you can remove the unwrap_or part. I think it can be:
```
#[pallet::type_value]
pub fn DefaultNodeCertifiers<T: Config>() -> Vec<T::AccountId> { vec![] }

....
pub type AllowedNodeCertifiers<T: Config> = StorageValue<_, Vec<T::AccountId>, OptionQuery, DefaultNodeCertifiers<T>>;
```

Reply:
Not really convinced we need this, tried it, to many compiler errors 

```
...
error[E0599]: the function or associated item `get` exists for struct `frame_support::pallet_prelude::StorageValue<_GeneratedPrefixForStorageAllowedNodeCertifiers<T>, std::vec::Vec<<T d

```